README.md

```diff
@@ -41,9 +41,37 @@ docker run --rm -v $(pwd):/go/src/${project} \
 - protoc-gen-gogoslick
 - protoc-gen-gogotypes
 - protoc-gen-gostring
-- protoc-gen-swagger
+- protoc-gen-swagger (**atlas-patch**)
 - protoc-gen-grpc-gateway
 - protoc-gen-validate
 - protoc-gen-govalidators
 - protoc-gen-doc
 - protoc-gen-gorm (**Infoblox Open**)
+
+## protoc-gen-swagger patch
+
+protoc-gen-swagger patch includes following changes:
```

Comment:
Why we have this doc here? Probably it should be [here](https://github.com/infobloxopen/grpc-gateway/tree/atlas-patch/protoc-gen-swagger) under README.

Reply:
it's more appropriate to place it here, because there is no point at all for someone to follow infobloxopen/grpc-gateway link. the only purpose of this branch is to be included in gentool, so gentool should be responsible of describing what it is about.

Reply:
Why no sense? It's our product which we develop, no matter it's derived from some other product. Storing documentation and actual product separately would complicate keeping documentation up-to-date while the product evolves.